content/docs/setup/kubernetes/quick-start-ibm/index.md

```diff
@@ -114,7 +114,13 @@ This guide installs the current release version of Istio.
     $ helm del istio --purge
     {{< /text >}}
 
-1. Delete the Istio custom resource definitions:
+    If your Helm version is less than 2.9.0, then you need to manually cleanup extra job resource before redeploy new version of Istio chart:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system delete job --all
+    {{< /text >}}
+
+1. If desired, delete the Istio custom resource definitions:
```

Comment:
@morvencao does Istio helm chart include post uninstall hooks can help delete those CRD? If so, I think we do not require the end user to delete those CRDs manually?

Reply:
@gyliu513 There is already PR that cleans up CRDs in post-delete hook: https://github.com/istio/istio/pull/8065

Reply:
I see, thanks @morvencao , then I think we can leave it as @linsun 's proposal here and update the document when istio/istio#8065 got merged.